Update School 2021 page

jme52 opened the pull request.

jme52 committed:
School 2021: add backlink from Lectures page.

jme52 committed:
Update School 2021 page with links to RTD and Siesta Mobile.

jme52 committed:
School 2021 is now in the past - this updates the events index.

albgar commented:
Sorry, I had not seen, or had forgotten, this PR!!!
Today I had updated independently a few things in the .md file and also made links to RTD from the Documentation section. I have taken from here the links to the VM. Changes already merged into master.
I close this now.  

Please accept the invitation to be a "maintainer" of the project, so that you can also merge things.

jme52 committed:
Merge master and pending changes to 2021 School pages.

jme52 committed:
School 2021: fix broken links to RTD.

The activation of multiple versions in RTD disabled the redirection to "latest".
This broke all the links to the Siesta RTD pages in the School 2021 pages.

This commit fixes those links by specifying the appropriate RTD version,
"siesta-2021", and language.

jme52 commented:
Hello,

I reopened this PR because (a) there were some changes that still not had been incorporated to master, and (b) there was a collection of related links that were broken and needed to be fixed.

I think all the changes are straightforward, so I'll probably merge later tonight.
